stdlib/public/Darwin/Dispatch/Block.swift

```diff
@@ -39,6 +39,7 @@ public struct DispatchWorkItemFlags : OptionSet, RawRepresentable {
 // used in the new runtime. _TtC8Dispatch17_DispatchWorkItem is the
 // mangled name for Dispatch._DispatchWorkItem.
 @available(macOS 10.10, iOS 8.0, *)
+@dynamicCallable
```

Comment:
I think this usecase calls for [SE-0253]'s static callables, not the dynamicCallable mechanism.

[SE-0253]: https://github.com/apple/swift-evolution/blob/master/proposals/0253-callable.md